### ISSUE TITLE
Network compressed POST connection error：Connection refused

### ISSUE THREAD

**Issue by GrowUpBoy** [googlesamples/google-services#352]
### Step 1: Are you in the right place?

  * For issues or feature requests related to the code **in this repository** file a Github issue.
  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/) tagged appropriately.

### Step 2: Describe your environment

  * Device: _____HAWEI-AL10
  * OS version: _____android
  * SDK version: _____play-services-analytics:11.0.4
  * [Android] Google Play Services version: _____3.0  
  
### Step 3: Describe the problem:
Log  content :
D/GAv4: Hit delivery requested: ht=1516072101831, _s=60, _v=ma11.0.20, a=1460009196, aid=com.condenast.gq24, an=gq24, av=1.0, cid=1f335075-01bb-4a87-8560-5a8cee982cc1, ea=Share_NO_proxy , ec=Action, ev=13, sr=1080x1812, t=event, tid=UA-112388999-1, ul=zh-cn, v=1
**01-16 11:08:21.878 1037-1087/com.condenast.gq24 D/GAv4: Hit sent to the device AnalyticsService for delivery
01-16 11:08:26.221 1130-30894/? W/GAv4-SVC: Network compressed POST connection error: java.net.ConnectException: Connection refused**

I don't know why the log display connection refused 
at the google analytics web interface , I cant see any data   
  
#### Observed Results:
D/GAv4: Hit delivery requested: ht=1516072101831, _s=60, _v=ma11.0.20, a=1460009196, aid=com.condenast.gq24, an=gq24, av=1.0, cid=1f335075-01bb-4a87-8560-5a8cee982cc1, ea=Share_NO_proxy , ec=Action, ev=13, sr=1080x1812, t=event, tid=UA-112388999-1, ul=zh-cn, v=1
**01-16 11:08:21.878 1037-1087/com.condenast.gq24 D/GAv4: Hit sent to the device AnalyticsService for delivery
01-16 11:08:26.221 1130-30894/? W/GAv4-SVC: Network compressed POST connection error: java.net.ConnectException: Connection refused**
  
#### Expected Results:

at the google analytics web interface , I can see any action data

**Comment by samtstern** [comment]
@GrowUpBoy thanks for reporting this, I don't know what causes this error so I have filed a bug internally with the Analytics engineering team to follow up.

**Comment by GrowUpBoy** [comment]
@samtstern Thank you for your help, and look forward to getting a reply soon.

**Comment by lbarrettanderson** [comment]
@GrowUpBoy 

Please try navigating to https://ssl.google-analytics.com/batch in the web browser on the mobile device you're testing with. Do you see a black screen with a single white pixel? Or something else?

**Comment by GrowUpBoy** [comment]
@lbarrettanderson 
I can open  https://ssl.google-analytics.com/batch page in my mobile device , and see a image that size is 1*1 pixel....  Does this mean I can access the Google anayltics server ?  but , at the google analytics web interface , I can't see any data . What should I do ?  lbarrettanderson , I need more advice .  Thank you for your help, and look forward to getting a reply soon.

**Comment by lbarrettanderson** [comment]
Thanks for trying that. Yes, that means you're able to access the proper server from the device itself (so this rules out firewall issues or captive portal issues).

I've never seen this behavior as a side effect of permission issues, but can you verify that you have internet permissions enabled on the app?

**Comment by GrowUpBoy** [comment]
@lbarrettanderson 
yes , I added Internet permissions at AndroidManifest.xml , and Be able to ensure that mobile devices are able to access the network .

**Comment by lbarrettanderson** [comment]
As a next step, can you try to make an https request for https://ssl.google-analytics.com/batch *from code within your app* ?
